network-api/networkapi/wagtailpages/templates/wagtailpages/primary_page.html

```diff
@@ -18,7 +18,7 @@
 
   <div class="container cms">
     <div class="row my-1 d-flex align-items-center">
-      <div class="{% if page.specific.narrowed_page_content %}col-md-8 offset-md-2{% else %}col-12{% endif %}">
+      <div class="{% if page.specific.narrowed_page_content %}col-lg-8 offset-lg-2{% else %}col-12{% endif %}">
```

Comment:
```suggestion
      <div class="pb-5 {% if page.specific.narrowed_page_content %}col-lg-8 offset-lg-2{% else %}col-12{% endif %}">
```
@mmmavis, I checked with @beccaklam and we should add `48px` of spacing between the body content and the donate banner to match the other templates.